GoogleForCreators/web-stories-wp
# Add video trimming quick action

## Context
Adds quick action menu for the Video trimming feature (currently behind a feature flag).


## Summary



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions
1. Enable Video trimming feature flag.
2. Add a video element
3. Verify it has the Trim quick action which enters the Trim mode.
4. Set the video as background.
5. Verify that now as well has the Trim quick action which enters the Trim mode.





## Checklist




---



Fixes #8878
